### DISCUSSION TITLE
Не запускается Tanks Blitz

### DISCUSSION THREAD

**Discussion by person10301** [dovecoteescapee/ByeDPIAndroid#139] [General]
![Screenshot_20240903-122508_Tanks Blitz](https://github.com/user-attachments/assets/94c99c7e-ea21-4afe-90f8-0e518c6f5849)
Пробовал разные настройки, до определенного обновления всё работало

**Comment by krlvm** [comment]
Нужно добавить приложение в список исключений, это станет возможно после #132

**Reply by person10301** [reply]
Ждём обновление значит
